src/files/modals/remove-modal/RemoveModal.js

```diff
@@ -42,13 +43,15 @@ const RemoveModal = ({ t, tReady, onCancel, onRemove, files, foldersCount, files
         <div className='charcoal w-90 center tl'>
           <p>{t(`removeModal.description${context}`, { count })}</p>
           { isLocallyPinned && (<div>
-            <input type="checkbox" className="mr1" id="removeLocalPin" name="removeLocalPin" checked={shouldRemoveLocalPin} onChange={handleLocalPinRemoval}/>
-            <label htmlFor="removeLocalPin">{t('removeModal.checkboxRemoveLocalPin')}</label>
+            <Checkbox className="mr1" name="removeLocalPin" checked={shouldRemoveLocalPin} onChange={handleLocalPinRemoval}
+              label={t('removeModal.checkboxRemoveLocalPin')} aria-label={t('selectAllEntries')}
```

Comment:
@rafaelramalho19 `selectAllEntries` is missing, is this an error? would it be ok to use same key as in `label` ?

Reply:
Ups, good catch that was a copy pasterino error, we don't need an aria-label because we already have the label